run/force_sync.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
+
+printf "Forcing archiveloop to sync by pretending to take archive host offline and back online.\n"
+printf "Setting archive unreachable.."
+if ! touch /tmp/archive-is-unreachable; then
```

Comment:
"then" goes on the next line

Reply:
It works here too, but if you prefer the longer convention, I can adapt.